src/styles/modules/_menu.scss

```diff
@@ -9,7 +9,7 @@
     position: relative;
     display: inline-block;
     color: $menu-link;
-    font-size: 0.75rem;
+    font-size: 1.125rem;
```

Comment:
Change in font size effects desktop and tablet view as well.
Make sure only mobile menu font-size should be changed